dashboard/lib/sample_data.rb

```diff
@@ -0,0 +1,181 @@
+include FactoryGirl::Syntax::Methods
+
+class SampleData
+  TEST_ACCOUNT_PASSWORD = "00secret".freeze
+  @@prng = nil
+
+  # Returns a seeded random number generator for consistent test data
+  def self.prng
+    @@prng ||= Random.new(0)
+    @@prng
```

Comment:
this line is unnecessary. Line 9 will evaluate to this anyway:

```
irb(main):002:0> defined? foo
=> nil
irb(main):003:0> foo ||= 'bar'
=> "bar"
irb(main):004:0> foo
=> "bar"
irb(main):005:0> foo = 'another value'
=> "another value"
irb(main):006:0> foo ||= 'bar'
=> "another value"
```

Reply:
Thx, changed